media-gfx/rawtherapee/rawtherapee-5.6.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic gnome2-utils toolchain-funcs xdg-utils
+
+MY_P=${P/_rc/-rc}
+DESCRIPTION="A powerful cross-platform raw image processing program"
+HOMEPAGE="http://www.rawtherapee.com/"
+SRC_URI="http://rawtherapee.com/shared/source/${MY_P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="openmp"
+
+RDEPEND="dev-libs/expat
+	dev-libs/libsigc++:2
+	media-libs/lcms:2
+	media-libs/lensfun
+	media-libs/libcanberra[gtk3]
+	media-libs/libiptcdata
+	media-libs/libpng:0
+	media-libs/tiff:0
+	sci-libs/fftw:3.0
+	sys-libs/zlib
+	virtual/jpeg:0
+	x11-libs/gtk+:3"
+DEPEND="${RDEPEND}
+	app-arch/xz-utils
+	dev-cpp/gtkmm:3.0
+	gnome-base/librsvg
+	virtual/pkgconfig"
```

Comment:
If you do bump EAPI to 7, this is then a BDEPEND dependency instead.

Reply:
That went a bit wrong :) 
```
RDEPEND="dev-libs/expat
	dev-libs/libsigc++:2
	media-libs/lcms:2
	media-libs/lensfun
	media-libs/libcanberra[gtk3]
	media-libs/libiptcdata
	media-libs/libpng:0
	media-libs/tiff:0
	sci-libs/fftw:3.0
	sys-libs/zlib
	virtual/jpeg:0
	x11-libs/gtk+:3"
DEPEND="${RDEPEND}
	app-arch/xz-utils
	dev-cpp/gtkmm:3.0
	gnome-base/librsvg"
BDEPEND="virtual/pkgconfig"
```
would be the correct form. I wasn't sure about xz-utils, if their binaries just get executed and no linking happens or no header files needed, then it's a BDEPEND too. 

BDEPEND can be a bit hard to understand, but https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html#build-time-dependencies-split-into-depend-and-bdepend tries to explain it. It's important to note that BDEPEND is a split off from DEPEND, thus can "never" inherit contents from RDEPEND or DEPEND.

Reply:
@juippis for some reason github does not send me emails for comments, please mention me or the risk is that I lose the comment forever
I've checked the xz-utils dep and it's not used at compile time (except from portage when unpacking) also a runtime it's only a dep of gtk not directly linked by rawtherapee. Has been removed
The change you suggest is now applied